portal/code/cx-portal/src/components/pages/Appstore/index.tsx

```diff
@@ -18,19 +17,31 @@ const Appstore = () => {
 
   return (
     <main className="Appstore">
-      <input
+      <SearchInput
+        margin="normal"
         autoFocus={true}
         value={searchParams.get('filter') || ''}
         onChange={(event) =>
           setSearchParams({ filter: event.target.value }, { replace: true })
         }
       />
       <nav>
-        {apps
-          .filter((app: App) => filter.test(app.name))
-          .map((app: App) => (
-            <AppCard key={app.id} app={app} />
-          ))}
+        <AppCards
+          items={apps
+            .filter(
+              (app: App) => filter.test(app.title) || filter.test(app.title)
```

Comment:
Es wird zweimal auf `app.title` getestet

Reply:
right - that should've been subtitle